lib/libzstd.pc.in

```diff
@@ -3,8 +3,9 @@
 #   BSD 2-Clause License (http://www.opensource.org/licenses/bsd-license.php)
 
 prefix=@PREFIX@
-libdir=@LIBDIR@
-includedir=@INCLUDEDIR@
+exec_prefix=${prefix}
```

Comment:
This is a completely moot point as in OpenWrt, the prefixes are overriden anyway.

In most of the pkgconfig files, /usr is specified instead of prefix:

```
mangix@DESKTOP-VIRSTQA:~/devstuff/openwrt/staging_dir/target-mips_24kc_musl/usr/lib/pkgconfig$ cat * | grep exec_prefix= | grep -c /usr
46
mangix@DESKTOP-VIRSTQA:~/devstuff/openwrt/staging_dir/target-mips_24kc_musl/usr/lib/pkgconfig$ cat * | grep exec_prefix= | grep -c \$\{
5
```

Again, completely moot point.